src/main/kotlin/com/hedvig/rapio/insurancecompanies/InsuranceCompaniesController.kt

```diff
@@ -0,0 +1,36 @@
+package com.hedvig.rapio.insurancecompanies
+
+import com.hedvig.libs.logging.calls.LogCall
+import com.hedvig.rapio.externalservices.underwriter.Underwriter
+import com.hedvig.rapio.insurancecompanies.dto.InsuranceCompanyDto
+import com.hedvig.rapio.insurancecompanies.dto.toDto
+import com.neovisionaries.i18n.CountryCode
+import feign.FeignException
+import mu.KotlinLogging
+import org.springframework.http.ResponseEntity
+import org.springframework.security.access.annotation.Secured
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+private val logger = KotlinLogging.logger {}
+
+@RestController
+@RequestMapping("v1/insurance-companies")
+class InsuranceCompaniesController(
+    val underwriter: Underwriter
+) {
+
+    @GetMapping
+    @Secured("ROLE_DISTRIBUTION")
```

Comment:
I suggest we make this one available for all our partners :) 
```suggestion
    @Secured("ROLE_DISTRIBUTION", "ROLE_COMPARISON")
```